src/index.js

```diff
@@ -0,0 +1,42 @@
+const inquirer = require("inquirer");
+const utilities = require("./utilities");
+const writeReadme = require("./readme");
+const { questions, installQuestions, testQuestions } = require("./questions");
+
+const start = async () => {
+  const readmeAnswers = await inquirer.prompt(questions);
+  const instructionsArray = [];
+  const testArray = [];
+
+  if (readmeAnswers.installationIncluded) {
+    let active = true;
+    while (active) {
+      const installationInstructions = await inquirer.prompt(installQuestions);
```

Comment:
Same here - destructure the object:

`const {step, continue} = await inquirer.prompt(installQuestions)`